File: uri/1st Page/cp with c++/URI 1013.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    int a, b, c, mab, mabc;
    cin >> a >> b >> c;
    mab = ((a+b+abs(a-b))/2);
    mabc = ((mab+c+abs(mab-c))/2);
    cout << mabc << " eh o maior" << endl;

    return 0;
}

File: uri/1st Page/cp with java/src/com/URI1013.java
import java.util.Scanner;
import static java.lang.Math.abs;

public class URI1013 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        int mab = (a+b+abs(a-b))/2;
        int m = (mab + c + abs(mab - c))/2;
        System.out.println(m+" eh o maior");
    }
}
